observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -165,3 +181,154 @@ spec:
 ----
 
 . Verify that logs are visible in the Log section of the Observe tab in the OpenShift web console.
+
+=== Quick Start with OTEL
```

Comment:
Do we need to duplicate the whole section or would it be more economic to just list the _differences_? I can see that some of the issues I have marked in the new content are just copied from the existing steps (they also need to be changed there) and I wonder if we can avoid this work in the future. Also it might be more digestable for users.

Reply:
I took the reference from Documentation Considerations mentioned in https://issues.redhat.com/browse/LOG-5637 in Jira description.

"The [default installation procedure](https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-about.html#quick-start) as described in 4.17+/6.0+ stays for as it, because we still want users to use the viaQ model as the currently GA format.

A new installation procedure copied from the [one in 4.17+/6.0+](https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.0/log6x-about.html#quick-start) needs to be established account for the dataModel field set to OTel as described here: https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/logforwarding/outputs/redhat-managed-lokistack.adoc"

Please let me know if you have other thoughts.